packages/components/src/components/button/_mixins.scss

```diff
@@ -81,6 +81,17 @@
       inset 0 0 0 ($button-border-width + $button-outline-width) $ui-02;
   }
 
+  &:disabled:hover,
+  &:disabled:focus,
+  &:hover.#{$prefix}--btn--disabled,
+  &:focus.#{$prefix}--btn--disabled {
```

Comment:
Seems like
<img width="349" alt="Screen Shot 2020-04-02 at 12 34 38 PM" src="https://user-images.githubusercontent.com/11928039/78294277-f6c6b200-74de-11ea-8fd9-dd8bf4886fae.png">
 `Dropdown` is not getting the proper text/icon colors when disabled: